include/mrdocs/Config.hpp

```diff
@@ -104,6 +104,12 @@ class MRDOCS_DECL
         /// from CMakeLists.txt
         std::string cmake;
 
+        /// True if the compiler has to use just the system's standard library
+        bool useSystemStandardLib = false;
+
+        /// LibC++ include paths (private option, not exposed to the user)
+        std::vector<std::string> libCxxPaths;
```

Comment:
This needs to go to TestArgs, where private options go and where this is used. These configuration files are even going to be automatically generated in the future.

Then you'll need a way to pass this libCxxPaths to the generator without Config, but that's not a huge problem. Another place for private config variables is ConfigImpl, but I'm not sure it applies here. Another solution is to make it public because users will have access to the option anyway, and this comment won't stop them. 

Also, if this is public, we need to discuss the design in a way we don't need for private options. For instance, if this is just a standard library path, it's not a libCxxPath because it's supposed to work with any standard library.

Reply:
I actually prefer to keep this option and make it public. With well-set defaults, I believe offering flexibility to the user is beneficial. 
What do you think?

Reply:
No problem. 

If it's always a single path, it's just a string option then. Not a vector. And the variable name would not be in the plural: `libcxx` instead of `libcxxs`

Would this also work with other standard library implementation headers? I don't see why not. In that case, the option name can't be `libcxx`. We can use the same name compilers use for the standard library path.

Reply:
> No problem.
> 
> If it's always a single path, it's just a string option then. Not a vector. And the variable name would not be in the plural: `libcxx` instead of `libcxxs`

It should remain a vector to accommodate multiple paths.


> Would this also work with other standard library implementation headers? I don't see why not. In that case, the option name can't be `libcxx`. We can use the same name compilers use for the standard library path: `stdlib`.

I'll apply the name change to stdlib.

Reply:
> accommodate multiple paths

My question is whether this is ever multiple paths.

> I'll apply the name change to stdlib.

Yes. This makes more sense.

Reply:
> My question is whether this is ever multiple paths.

I need to add libC++ and some Clang headers, for example. I realized this after doing several tests. It is likely that some other path will be needed in the future.